From: Ben Hutchings <ben@decadent.org.uk>
Subject: Define netdev_features_t

This was done in upstream commit
c8f44affb7244f2ac3e703cab13d55ede27621bb, but we just want the type
definition for use by backported drivers rather than changing
everything to use it.
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -950,6 +950,8 @@
 						    u32 features);
 };
 
+typedef u32 netdev_features_t;
+
 /*
  *	The DEVICE structure.
  *	Actually, this whole structure is a big mistake.  It mixes I/O
